src/sentry/api/event_search.py

```diff
@@ -1418,6 +1418,7 @@ def get_json_meta_type(field_alias, snuba_type, function=None):
 
 
 FUNCTION_PATTERN = re.compile(r"^(?P<function>[^\(]+)\((?P<columns>[^\)]*)\)$")
+ALIAS_PATTERN = re.compile(r"(\w+)?(?!\d+)\w+$")
```

Comment:
I don't think this impacts this PR right now, but the aliases can now contain comparison operators which would not match this pattern.